packages/kbn-test/src/functional_test_runner/cli.ts

```diff
@@ -69,6 +69,10 @@ export function runFtrCli() {
         process.env.TEST_BROWSER_HEADLESS = '1';
       }
 
+      if (flags['ignore-tls-errors']) {
+        process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
```

Comment:
@elastic/kibana-operations what's the better way to make FTR work with self-signed cert? The current test fails on fetching Kibana status https://github.com/elastic/kibana/blob/d049acf4295c5faa49d6a5fe181b3d33675b5f97/packages/kbn-dev-utils/src/kbn_client/kbn_client_requester.ts#L96
I hardcoded this logic for testing purposes and it works for `scripts/functional_test_runner.js`. I want to hear your opinion on that, before I added something similar to `node scripts/functional_tests.js` to make it work on CI.

Reply:
I think the best option would probably be to extend the [`urlPartsSchema` in the functional test runner](https://github.com/elastic/kibana/blob/master/packages/kbn-test/src/functional_test_runner/lib/config/schema.ts#L30-L42), define the certs to use for communicating with the Kibana server, and then add support for specifying the certs to the `KbnClient` so that all requests to the server use those certs. If you can help me get thing setup locally I'd be happy to help with the changes.

Reply:
@spalger I actually played with this approach the last week. I hardcoded `kbn-dev-utils` certs in `KbnClientRequester`:
```js
const httpsAgent = new Https.Agent({
  cert: Fs.readFileSync(KBN_CERT_PATH),
  key: Fs.readFileSync(KBN_KEY_PATH),
  ca: Fs.readFileSync(CA_CERT_PATH),
  // rejectUnauthorized: false, << doesn't work either
});

const response = await Axios.request<T>({
  //...
  httpsAgent,
});
```
but as I wrote before I was getting: `ERROR Error: self signed certificate in certificate chain`. Seems the self-signed cert is the root of the problem. However, I don't want to add another cert in the repo.